src/components/fileUpload.js

```diff
@@ -229,7 +394,14 @@
 
     B.defineFunction('clearFileUpload', e => clearFiles(e));
 
-    return isDev ? <div className={classes.root}>{Control}</div> : Control;
+    return isDev ? (
+      <div>
```

Comment:
This can probably be a fragment?

Reply:
This one must be a div so that the component is selectable within the builder.

Reply:
Ah got it 👌